lrpar/src/lib/ctbuilder.rs

```diff
@@ -416,6 +421,15 @@ where
         self
     }
 
+    #[cfg(test)]
```

Comment:
Do we need to make this `doc(hidden)` too? I think it will show up in rustdoc otherwise (but I might be wrong).

Reply:
We shouldn't need to, by default rustdoc only documents the specified cfg values, or the ones specified by `cfg(doc)`

https://doc.rust-lang.org/rustdoc/advanced-features.html#cfgdoc-documenting-platform-specific-or-feature-specific-information